### PR TITLE
Rigidbody2D Bird Flying Controller

### LINKED ISSUE #1: Flying Bird
Create a controllable 2d object that moves up and down in fixed motion with a effect of gravity. 

*Recommended to use Rigidbody component with gravity set on.

### PR DESCRIPTION
**Pull request by Galaxy25**
Closes #1.

### CODE REVIEW

**Review by Galaxy25 — COMMENTED** [review]
Appears to be working

**Comment by BrandonPacewic** [comment]
Hey thanks for the changes, give me some time to look at your changes. Super excited! Feel free to ask me any questions! :bird:

**Comment by BrandonPacewic** [comment]
Auto response test

File: Assets/Scripts/Flying.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public Rigidbody2D rb;
+    float force = 10.0f;
+    bool hit = true;
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
```
Comment (BrandonPacewic) [inline Assets/Scripts/Flying.cs]: This comment is unnecessary and should always be removed.
```suggestion
```

File: Assets/Scripts/Flying.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Flying : MonoBehaviour
+{
+    public Rigidbody2D rb;
+    float force = 10.0f;
+    bool hit = true;
+    // Start is called before the first frame update
```
Comment (BrandonPacewic) [inline Assets/Scripts/Flying.cs]: Same as other comment.
```suggestion
```

File: Assets/Scripts/Flying.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Flying : MonoBehaviour
+{
+    public Rigidbody2D rb;
+    float force = 10.0f;
```
Comment (BrandonPacewic) [inline Assets/Scripts/Flying.cs]: Should these variables be private or public?

```suggestion
    private float force = 10.0f;
```

File: Assets/Scripts/Flying.cs [hunk 4]
```diff
@@ -0,0 +1,29 @@
+using System.Collections;
```
Comment (BrandonPacewic) [inline Assets/Scripts/Flying.cs:4]: Need to add our copyright to the top
```suggestion
// Copyright (c) TigardHighGDC
// SPDX-License SPDX-License-Identifier: Apache-2.0

using System.Collections;
```

File: Assets/Scripts/Flying.cs [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Flying : MonoBehaviour
+{
+    public Rigidbody2D rb;
+    float force = 10.0f;
+    bool hit = true;
+    // Start is called before the first frame update
+    void Start()
```
Comment (BrandonPacewic) [inline Assets/Scripts/Flying.cs]: Do we really need start?

File: Assets/Scripts/Flying.cs [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    {
+    }
+
+    // Update is called once per frame
+    void Update()
```
Comment (BrandonPacewic) [inline Assets/Scripts/Flying.cs]: Should be private
```suggestion
    private void Update()
```

**Review by BrandonPacewic — CHANGES_REQUESTED** [review]
Lets look at removing start and the update comment. Seems unnecessary.

**Review by BrandonPacewic — APPROVED** [review]
@Galaxy25 Looks good to me, although in the future I would like to only add code that we know works and can be ran.

Since you can't review your own PR just add a comment like:

**Approve** ✅

So I know that you are ok with the PR

**Comment by Galaxy25** [comment]
Approve

**Comment by BrandonPacewic** [comment]
I'm getting ready to merge and updating the change log - please @me if any other changes are pushed.

**Comment by BrandonPacewic** [comment]
Thanks for getting everyone started on this! I can't wait to get everything rolling this year 🚀

**Pull request merged**